index.js

```diff
@@ -50,6 +59,9 @@ module.exports = function installGit(options) {
                     }
                 });
             }
-        }
+        }, error => {
+            reject(new Error(`lambda-git failed to load: ${error}`));
+        });
```

Comment:
We shouldn't need to wrap a promise to manually call resolve/reject when we've already got a promise coming back from `lambdaBinRuntime.applyMinPack`. I think you can change this to something like:

```javascript
function installGit(options) {
  options = options || {}; // (skip the new Promise)
  ...
  return lambdaBinRuntime.applyMinPack(targetDirectory).then(() => {
    ...
    // Now normal returns work fine, no resolve():
    return {
      binPath: binPath
      ...
    };
  });
  // no error => reject() needed - promise errors propagate automatically
```

Reply:
Done

Reply:
Will push this change once I hear back from you on the other items.